Cython/Compiler/PyrexTypes.py

```diff
@@ -198,6 +198,8 @@ class PyrexType(BaseType):
     #  is_buffer             boolean     Is buffer access type
     #  is_pythran_expr       boolean     Is Pythran expr
     #  is_numpy_buffer       boolean     Is Numpy array buffer
+    #  is_initvar            boolean     Is a dataclass InitVar
+    #  is_classvar            boolean     Is a dataclass ClassVar
```

Comment:
I think these erequire context. Otherwise they'd read too generic when used in code.
```suggestion
    #  is_dataclass_initvar            boolean     Is a dataclass InitVar
    #  is_dataclass_classvar           boolean     Is a dataclass ClassVar
```